app/services/food_service.rb

```diff
@@ -0,0 +1,10 @@
+class FoodService
+  def self.search(query)
+    response = Faraday.get "https://api.nal.usda.gov/fdc/v1/foods/search?query=#{query}&pageSize=2&api_key=#{ENV['FOOD_API_KEY']}"
+    parse(response)
+  end
+
+  def private_class_method.parse(response)
```

Comment:
This is interesting syntax! I have not seen this before. Nice job extracting this api consumption functionality into a service.